lib/curator.js

```diff
@@ -141,9 +141,23 @@ bedrock.events.on('bedrock-express.configure.routes', function(app) {
     '/tasks/credentials/compose-identity',
     relaxedAuthenticate, acceptableContent('urlencoded'),
     parseForm, parseJsonData, function(req, res, next) {
+    var credentialQuery = req.body.query;
+
+    /*
+     * pubicKey has the following properties
+     * type: array that always contains 'CryptographicKey' and can also contain
+     * ‘EmphemeralCryptographicKey’ or ‘NewCryptographicKey’
```

Comment:
What is "NewCryptographicKey"? ... I don't think it's proper modeling to specify whether or not a key is new via its type. Instead, a check could be done to see if it's `id` has been set or not. If no `id` has been set, then we're dealing with a "new" key. If the type includes "EphemeralCryptographicKey" (typo btw on that line) then we know not to store it.